Yona/Yona/Others/Constants.swift

```diff
@@ -208,7 +208,8 @@ struct YonaConstants {
     }
     
     struct mobileDucthPrefix {
```

Comment:
Even better: Let's just do away with this struct. Given it's used only once, let's just put the constant where we use it, see below.